README.md

```diff
@@ -131,6 +131,14 @@ In addition, the following methods are available on the subscriber
 
 Close all established connections.
 
+#### `await subscriber.pause()`
+
+Pause consumption of new messages.
+
+#### `await subscriber.unpause()`
```

Comment:
let's rename this to `resume` to follow typical patterns used elsewhere (like node's streams)

Reply:
@nlf works for me 😁